CDCgov/prime-reportstream
# 10508: update schema pathing

This PR moves the FHIR->FHIR and FHIR->HL7 schemas into the classpath and then updates all the schema properties to be URIs.

Test Steps:
1. Run the smoketests
2. Verify that you can still use the prime CLI for
 - FHIR -> FHIR (sender and enrichment)
 - FHIR -> HL7
3. Run a FHIR report and HL7 report through the pipeline

## Changes
- Moves all the FHIR->FHIR|HL7 schemas into the classpath
- Updates all the schemas to reference URIs
- Adds a small function to convert to URIs while the settings are not updated

## Checklist

### Testing

### Process

## Linked Issues
- #10508 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #10510 

## Specific Security-related subjects a reviewer should pay specific attention to
